Description: fixed typo in iscsi_discovery usage()
Forwarded: https://github.com/open-iscsi/open-iscsi/pull/2
Author: Mark Karpeles <mark@hell.ne.jp>
Last-Update: 2013-10-09

---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

 utils/iscsi_discovery | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/iscsi_discovery b/utils/iscsi_discovery
index d07ecc2..dcb30ee 100755
--- a/utils/iscsi_discovery
+++ b/utils/iscsi_discovery
@@ -39,7 +39,7 @@ usage()
 	echo "Options:"
 	echo  "-p		set the port number (defualt is 3260)."
 	echo  "-d		print debugging information"
-	echo  "-t		set trasnpot (default is tcp)."
+	echo  "-t		set transport (default is tcp)."
 	echo  "-f		force specific transport -disable the fallback to tcp (default is fallback enabled)."
 	echo  "			force the transport specified by the argument of the -t flag."
 	echo  "-m		manual startup - will set manual startup (default is automatic startup)."
